docs/tutorials/setup-env.md

```diff
@@ -0,0 +1,165 @@
+
+Now you can quickly start a new DHIS2 instance by running the following command from your terminal: 
+
+```shell
+d2 cluster up <name>
+# or:
+d2 cluster up 2.35.0 --db-version 2.35 —-seed
+# this will populate a database with sample data
+```
+
+The command above uses the `d2 cluster` tool with the `up` command, the `--db-version` flag and the `--seed` flag to spin up a new DHIS2 2.35.0 instance seeded with data from the Sierra Leone [demo database](https://dhis2.org/demo).
+
+> If you want to test against different DHIS2 and database versions or build your own custom DHIS2 Docker image, please check the [DHIS2 CLI](https://cli.dhis2.nu/#/commands/d2-cluster) documentation for more information. 
+
+For more d2 cluster commands, run `d2 cluster --help`.
+
+### Check your new DHIS2 instance
+
+After the above command has completed you can check your newly created cluster that's running locally at [http://localhost:8080/](http://localhost:8080/). If DHIS2 is running, you should see the following page: 
+
+![Login Page](../assets/quickstart_guides/image-of-login.png)
+
+Sign in as `admin` (username) and `district` (password). If you can load DHIS2 in the browser, you are ready to create a new app. 
+
+### Stopping an instance
+
+The `down` command followed by the name of the cluster shuts down all Docker processes connected to that cluster.
+
+```shell
+d2 cluster down 2.35.0
+```
+
+## 3. Initialize a new DHIS2 app
+
+Now you are ready to use the `d2` CLI tool to create your first DHIS2 app. 
+
+The `d2` CLI provides a simple way of initializing a new app. We'll use the [d2-app-scripts init command](https://platform.dhis2.nu/#/scripts/init) to do that. 
+
+From your terminal, navigate to the where you want to create your project and then run the following command: 
+
+```shell
+d2 app scripts init <app-name>
+# <app-name> must be replaced with your app name.
+``` 
+We can now navigate into the newly created folder and start your application locally: 
+
+```shell
+yarn start
+```
+
+### Connecting your app to the DHIS2 instance 
+
+After running `yarn start`, a web-browser window should have popped up greeting you with a login dialog at [http://localhost:3000](http://localhost:3000). 
+
+Enter your DHIS2 server URL and the username and password of the default admin user (as shown below): 
+
+```
+server: http://localhost:8080
+username: admin
+password: district
+```
+
+You will then see your newly initialized application greeting screen like this one 👇 
+
+![Sign In](./assets/sign-in-app.gif)
+
+**NOTE**: If you're running into some Cross-Origin Resource Sharing [(CORs)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) policy issues when trying to connect to your application, please check the guide on [how to debug common development errors](../guides/debug-instance). 
+
+### Set up code-style with DHIS2 style
```

Comment:
I really like recommending this, and we might want to add it to the `init` template, but maybe it should be in a separate guide to keep this totrial laser-focused on setting up a new app?

Reply:
Yes, you're right! I created a 'how-to-guide' for this one instead